test-suite-tck-serde/src/test/java/io/micronaut/serde/tck/tests/serde/SerializationSerdeSuite.java

```diff
@@ -1,10 +1,14 @@
 package io.micronaut.serde.tck.tests.serde;
 
+import org.junit.platform.suite.api.ExcludeClassNamePatterns;
 import org.junit.platform.suite.api.Suite;
 import org.junit.platform.suite.api.SelectPackages;
 import org.junit.platform.suite.api.SuiteDisplayName;
 
 @Suite
+@ExcludeClassNamePatterns(
+    "io.micronaut.serde.tck.tests.bytebuffer.ByteBufferNativeTest"
```

Comment:
This test is excluded. I don't see how to easily port this port this part of the serializer
https://github.com/FasterXML/jackson-databind/blob/2.16/src/main/java/com/fasterxml/jackson/databind/ser/std/ByteBufferSerializer.java#L27-L35

Reply:
since youre copying the data anyway, you can just copy it for direct buffers too. but ideally there should be overloads for encodeBinary for ByteBuffers.